README.md

```diff
@@ -1,2 +1,168 @@
-# hello-rails-react
-Implement a connection between a Ruby on Rails back-end and React front-end.
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello React Rails <a name="about-project"></a>
+
+**Hello React Rails** is an implementation of Ruby on Rails and React-Redux with webpack
+
+- It shows a random greeting
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>HTML</li>
+   <li>Bootstrap</li>
+   <li>React</li>
+   <li>Redux</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- It shows a random greeting
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Currently, I don't have a live demo of this project. I will let you know when it is available ASAP!
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Getting Started -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To run the project locally, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+
+- Install [Git](https://git-scm.com/)
+- Install [Ruby](https://www.ruby-lang.org/en/)
+- Install Rails gem `gem install rails`
+- Install a code editor, We suggest [VsCode](https://code.visualstudio.com/)
+
+### Setup
+
+Clone the repo into your local machine:
+
+```bash
+git clone git@github.com:OsvaldoBC/hello-rails-react.git
+```
+
+Install the dependencies using bundler:
+```bash
+bundle install
+```
```

Comment:
- [ ] Also before installing bundles, I thing it would be better if you add a command of how to access the cloned directory. This will ensure that user(s) has/have all information without getting an error. You can update it to something like this below:
```
### Install

Install the dependencies using bundle:
```bash
cd hello-rails-react

bundle install
```